patterns/atoms/global/colors/_colors.scss

```diff
@@ -0,0 +1,28 @@
+
+.colors__color1 {
```

Comment:
For the record...I understand how colors work in css and that they are not supposed to be part of the names of classes. The classes I'm using above do not have colors in their names and they were not meant as pseudonyms for names that would have color names in them. They are merely some temporary classNames which can be applied style from a project's main styles - which could of course include variables.